### PR TITLE
Refactoring css

### PR DESCRIPTION
**Pull request by shiba4life**
buttons are in their own file
variables.scss has colors
box-shadow is now a mix-in
removed inline css for tags

### CODE REVIEW

File: clientapp/styles/button.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+.button{
+  background: url(/static/button-blue.svg) no-repeat;
+  @extend .button-base
+}
+.button.secondary{
+  background: url(/static/button-grey.svg) no-repeat;
+  @extend .button-base
+}
+.button.alert{
```
Comment (ZeeJab) [inline clientapp/styles/button.scss]: is this meant to be `.button-alert { }` ?

Comment (shiba4life) [inline clientapp/styles/button.scss]: yeah, that's probably better, don't want it conflicting with other alert classes.  I'll fix it.

**Comment by ZeeJab** [comment]
this lgtz for merging @stenington. There's still very specific styles being made, but it's ok for now I think.

File: clientapp/styles/button.scss (hunk 1 above)
Comment (shiba4life) [inline clientapp/styles/button.scss]: Any thoughts on what the button for the signout dropdown should be called?  button-signout? button-menu?

**Comment by stenington** [comment]
Cool, merging! :+1:

**Comment by shiba4life** [comment]
Awesome thanks!

**Pull request merged**